unison-cli/src/Unison/CommandLine/Welcome.hs

```diff
@@ -166,7 +166,7 @@ authorSuggestion :: P.Pretty P.ColorText
 authorSuggestion =
   P.newline
     <> P.lines
-      [ P.wrap "📜 🪶 You might want to set up your author information next.",
+      [ P.wrap "📜 You might want to set up your author information next.",
```

Comment:
Can we not use the 🖊️ character anymore?

Reply:
I haven't tested on GHC 9.2.4. On GHC 9.0.2, no, which was weird. But I didn't investigate too closely because I thought putting two emojis in a row here was a mistake, anyway.

Reply:
Should I try putting it back?

Reply:
I dunno, I guess it's supposed to be writing on paper, not sure if the paper alone makes sense. @rlmark ? :)